signin.js

```diff
@@ -0,0 +1,84 @@
+const emailInput= document.querySelector('.email-input');
```

Comment:
보통 `=` 좌우로 공백을 넣습니다
전체적으로 확인 후 수정해주세요! 

```suggestion
const emailInput =  document.querySelector('.email-input');
```